tools/accuracy_checker/accuracy_checker/annotation_converters/cmu_panoptic.py

```diff
@@ -0,0 +1,122 @@
+"""
+Copyright (c) 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import numpy as np
+from pathlib import Path
+
+import cv2
+
+from ..utils import read_json, check_file_existence
+from ..representation import PoseEstimation3dAnnotation
+from .format_converter import DirectoryBasedAnnotationConverter, ConverterReturn
+
+
+class CmuPanopticKeypointsConverter(DirectoryBasedAnnotationConverter):
+    __provider__ = 'cmu_panoptic_keypoints'
+    annotation_types = (PoseEstimation3dAnnotation,)
+
+    def configure(self):
```

Comment:
no need in this method, if you look on configure method of `DirectoryBasedAnnotattionConverter`, it already do the same thing.

Reply:
Thanks, cleaned.